src/tests/integration/test_pipeline_integration.py

```
@@ -0,0 +1,195 @@
+"""Integration tests for the VeriFact pipeline."""
+
+import pytest
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from src.tests.fixtures.claims import POLITICAL_CLAIMS, HEALTH_CLAIMS, SAMPLE_TEXTS
+from src.tests.fixtures.evidence import POLITICAL_EVIDENCE, HEALTH_EVIDENCE
+from src.tests.fixtures.verdicts import POLITICAL_VERDICTS, HEALTH_VERDICTS
+
+from src.verifact_agents.claim_detector import Claim
```

Ruff (0.11.9)

`src.verifact_agents.claim_detector.Claim` imported but unused

Remove unused import: `src.verifact_agents.claim_detector.Claim`

(F401)